services/core-api/app/api/mines/permits/permit_amendment/models/permit_amendment.py

```diff
@@ -85,7 +85,7 @@ class PermitAmendment(SoftDeleteMixin, AuditMixin, Base):
     def issuing_inspector_name(self):
         title = "Inspector of Mines"
 
-        #with i had null propogation
+        #with i had null propagation
```

Comment:
I don't understand this comment. What does it mean?

Reply:
Have no idea, there was a typo that I fixed